### DISCUSSION TITLE
How do you know if your proof verification times are too long via the GUI?

### DISCUSSION THREAD

**Discussion by number435398** [Chia-Network/chia-blockchain#6942] [General]
How do you know if your proof verification times are too long via the GUI? How do you know if you failed to win a block because of your proof verification times being too long as opposed to simply not having the best available proof?

**Comment by laisangbum** [comment]
I install chiadog to notify if the time took too long, or farming have problem

**Reply by number435398** [reply]
What helps with those times, either way?

**Comment by joelrb** [comment]
Open debug.log. Search for qualities. Look for entry like this:

2021-06-02T23:56:24.403 harvester chia.harvester.harvester: WARNING Looking up qualities on D:\Folder\plots\plot-k32-xxxx.plot took: 7.8580029010772705. This should be below 5 seconds to minimize risk of losing rewards.
